lib/pages/home/wallet/wallet.dart

```diff
@@ -57,12 +57,19 @@ class WalletPage extends StatelessWidget {
     );
   }
 
+  var rowCounter = 1;
```

Comment:
Don't use non-final variables in Stateless Widgets. Instead of mapping the currencies to _buildCurrencyRow like is done in the build method, use List.generate and grab the correct currency using the index. This index can then be used to determine the color of the row.

Reply:
I'll look at rewriting this. This is my first time using Flutter so I'm not familiar with List.generate yet.

Reply:
I've rewritten this now using List.generate.